### PR TITLE
Comment, custom code, and constant value block bug fixes

### PR DESCRIPTION
**Pull request by MatzElectronics**
Minimum fixes required to address all console warnings and errors sourced by block definitions when uploading demo project ID 2075 into solo.

### CODE REVIEW

File: src/blockly/generators/propc/variables.js [hunk 1]
```diff
@@ -229,9 +232,9 @@ Blockly.Blocks.array_get = {
 
         // Init the array name to a default if there are no existing
         // array variable blocks
-        if (v_check) {
+        //if (v_check) {
```
Comment (zfi) [inline src/blockly/generators/propc/variables.js]: Same issue here, where commented out code should be removed. If we're not sure about it, the code shouldn't be in the PR.

File: src/blockly/generators/propc/base.js [hunk 2]
```diff
@@ -2421,9 +2423,9 @@ Blockly.Blocks.constant_value = {
                 v_check = false;
             }
         }
-        if (v_check) {
+        //if (v_check) {
```
Comment (zfi) [inline src/blockly/generators/propc/base.js]: If the code isn't needed, let really remove it.

File: src/blockly/generators/propc/base.js [hunk 3]
```diff
@@ -1288,8 +1287,11 @@ Blockly.Blocks.comment = {
 
 Blockly.propc.comment = function () {
     var text = this.getFieldValue("COMMENT_TEXT");
-
-    return '// ' + text.replace(/,+\s*$/g, '') + '\n';
+    var code = '';
```
Comment (zfi) [inline src/blockly/generators/propc/base.js]: It looks like this code is being fail-safe. Consider replacing
```
    var code = '';
    if (text) {
        code = '// ' + text.replace(/,+\s*$/g, '') + '\n';
    }
    return code;
```
with this:
```
return (text) ? '// ' + text.replace(/,+\s*$/g, '') + '\n' : '';
```
It eliminates a variable and is still unambiguous.

Comment (MatzElectronics) [reply src/blockly/generators/propc/base.js]: @MatzElectronics reminder to do this.

**Pull request merged**